service/create/launch_configuration.go

```diff
@@ -91,21 +93,23 @@ func (s *Service) createLaunchConfiguration(input launchConfigurationInput) (boo
 		return false, microerror.MaskAny(err)
 	}
 
-	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix)
+	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix, securityGroupID)
```

Comment:
The SG ID changes every time, since I was deleting the SG as part of cluster deletion.